contrib/epee/include/net/net_helper.h

```diff
@@ -562,11 +553,9 @@ namespace net_utils
 	protected:
 		boost::asio::io_service m_io_service;
 		boost::asio::ip::tcp::socket m_socket;
-		int m_connect_timeout;
-		int m_reciev_timeout;
 		bool m_initialized;
 		bool m_connected;
-		boost::asio::deadline_timer m_deadline;
+		boost::asio::steady_timer m_deadline;
```

Comment:
I could also do `expires_from_now(boost::posix_time::milliseconds(timeout.count()))`, so the timer type could remain the same - reducing the number of lines touched by the patch.